typescript/cli/src/tests/cosmosnative/core/core-deploy.e2e-test.ts

```diff
@@ -52,25 +56,23 @@ describe('hyperlane cosmosnative core deploy e2e tests', async function () {
     assert(chainMetadata.gasPrice, 'gasPrice not defined in chain metadata');
 
     const wallet = await DirectSecp256k1Wallet.fromKey(
-      Buffer.from(HYP_KEY, 'hex'),
+      new Uint8Array(Buffer.from(HYP_KEY, 'hex')),
       'hyp',
     );
 
-    signer = await SigningHyperlaneModuleClient.connectWithSigner(
-      chainMetadata.rpcUrls[0].http,
+    signer = await CosmosNativeSigner.connectWithSigner(
+      chainMetadata.rpcUrls.map((rpc) => rpc.http),
       wallet,
       {
-        gasPrice: GasPrice.fromString(
-          `${chainMetadata.gasPrice.amount}${chainMetadata.gasPrice.denom}`,
-        ),
+        metadata: chainMetadata,
       },
     );
 
-    initialOwnerAddress = signer.account.address;
+    initialOwnerAddress = signer.getSignerAddress();
   });
 
   describe('hyperlane cosmosnative core deploy', () => {
-    it('should create a core deployment with the signer as the mailbox owner', async () => {
+    it('1. should create a core deployment with the signer as the mailbox owner', async () => {
```

Comment:
i think that we can avoid adding numbers to the test description as we will need to update the indices if we add another test in between the existing ones